### PR TITLE
Fixed a couple of spelling errors

### PR DESCRIPTION
**Pull request by johanhelsing**

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

File: docs/api/extras/geometries/TextGeometry.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		<h3>[name]([page:String text], [page:Object parameters])</h3>
 		<div>
 		text — The text that needs to be shown. <br />
-		parameters — Object that can contain the following parameters.
+		parameters — Object that can contains the following parameters.
```
Comment (fraguada) [inline docs/api/extras/geometries/TextGeometry.html]: This line 23 was correct to begin with.  Should have a semicolon if it will refer to a list.

**Comment by johanhelsing** [comment]
@CodingFabian Yeah, you're right. It was correct to begin with.

I somehow messed up and managed to read it as "Object that contain the following parameters", which would have been incorrect.

File: docs/api/extras/geometries/TextGeometry.html (hunk 1 above)
Comment (WestLangley) [inline docs/api/extras/geometries/TextGeometry.html]: @fraguada is correct.

**Comment by mrdoob** [comment]
Could someone do another PR? 😅

**Pull request merged**